controllers/apply/form-router-next/lib/file-uploads.js

```diff
@@ -19,6 +20,22 @@ const s3 = new AWS.S3({
     region: 'eu-west-2'
 });
 
+const ClamScan = new NodeClam().init({
+    debug_mode: false,
+    scan_recursively: false,
+    clamdscan: {
+        socket: config.get('clamdscan.socket'),
+        timeout: config.get('clamdscan.timeout'),
+        local_fallback: true,
+        path: config.get('clamdscan.path'),
```

Comment:
I'd probably set these config.get calls to be `process.env.CLAMDSCAN_PATH || config.get('clamdscan.path')` etc. so we can set these values locally. Using `development.json` for this doesn't really work as local config will be specific to our individual set ups.